src/components/Footer/icons.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from 'react';
+import { chakra, VisuallyHidden } from '@chakra-ui/react';
+import { FaInstagram, FaTwitter, FaYoutube } from 'react-icons/fa';
+
+/* Social Media Button Props */
+interface SocialProps {
+  children?: ReactNode;
+  label: string;
+  href: string;
+  _hover: object;
+}
+
+/* Social Media Buttons */
+export function SocialButton(props: SocialProps) {
+
+  const { children, label, href, _hover } = props;
+
+  return (
+    <chakra.button
```

Comment:
@kevinjung13 
another approach can be to use the `Link` component from chakra use the props `isExternal` and `href`
This component should wrap the Button

Reply:
@jeromevvb Ah perfect, just tried and it works well so ill do that. Thanks!